src/app.py

```diff
@@ -8,7 +8,6 @@
 
 tsunami_df=pd.read_csv('data/processed/tsunami-events.csv')
 
-years=tsunami_df['year'].dropna().unique()
 countries=tsunami_df['country'].dropna().unique()
```

Comment:
@Jacq4nn we are using tsunami_events everywhere instead of tsunami_df. Can this be changed for consistency?

Reply:
Will work on that